packages/i18n/src/messages/apiMessages.ts

```diff
@@ -232,6 +232,21 @@ export const apiMessages: { [key: string]: MessageDescriptor } = defineMessages(
       description:
         "Error message that the pool advertisement must have publishing group filled.",
     },
+    AssessmentStepMissingSkills: {
+      defaultMessage:
+        "Please ensure the assessments are connected to one or more skills",
+      description:
+        "Error message that the pool advertisement assessments are lacking a skill",
+      id: "rgxNWX",
+    },
+    PoolSkillsWithoutAssessments: {
+      defaultMessage:
+        "Please ensure the advertisement's skills are connected to one or more assessments",
+      description:
+        "Error message that the pool advertisement skills are lacking an assessment",
+      id: "ZWEzaw",
+    },
+
```

Comment:
Tiny nitpick that you probably were not aware of but, the copy editors have recommended we stop using "Please" because it implies it is a request. Though, it is not a request, it is something the user _must_ do. Could we rephrase these to be statements rather than requests?

- Each assessment must include one or more skill for evaluation
- Each skill must be included in an assessment

Reply:
Politeness, dial down in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8546/commits/b22057d0b835a0e4494380845dfd645b6bc1ea98

Reply:
haha, that is an apt description 🤣